tests/hilti/codegen/library-version.hlt

```diff
@@ -0,0 +1,7 @@
+# @TEST-EXEC: hiltic -x foo %INPUT
+# @TEST-EXEC: grep -q _hlt_hlto_library_version foo___linker__.cc
+# @TEST-EXEC: grep -q _hlt_hlto_bind_to_version foo___linker__.cc
```

Comment:
The original issue was that hand-built HLTO files would not load due to us rejecting them because of missing version info, maybe add that to the test as well since we didn't have an end-to-end test for that.

```suggestion
# @TEST-EXEC: hiltic -x foo %INPUT
# @TEST-EXEC: grep -q _hlt_hlto_library_version foo___linker__.cc
# @TEST-EXEC: grep -q _hlt_hlto_bind_to_version foo___linker__.cc
#
# A hand-built HLTO file has all symbols needed to load correctly.
# @TEST-EXEC: $(spicy-config --cxx --cxxflags-hlto --ldflags-hlto --debug) -o foo.hlto foo*cc
# @TEST-EXEC: hiltic -j foo.hlto
```